### DISCUSSION TITLE
How to use trec_eval?

### DISCUSSION THREAD

**Discussion by ls0218** [irgroup-classrooms/dis17-2021#30] [General]
Hello,

we have downloaded the file trec_eval_win32.zip but we cannot execute it. 

We still do not really feel comfortable how the evaluation works and how we use or rather adapt the simple-run.py. 

We are also wondering whether we have to create a new collection and index the data again everytime we want to change something?

Is it possible to show us how it works on Friday? Maybe it is possible to explain the workflow again?

We feel familiar with the theoretical part but we are struggling with the implementation.

Thank you!

**Comment by phschaer** [comment]
Dear Ms. Schulz, 

did you include the DLL file included in the ZIP in the same folder as the EXE file? The Windows version needs EXE and DLL files in the same folder to work.

**Reply by ls0218** [reply]
Dear Mr. Schaer,

Thank you for your answer. 
I have the DLL file in the folder but it does not work. So if I click on the EXE file, there is a popup coming up but then it is closing directly.

**Reply by phschaer** [reply]
Please keep in mind: this is a tool for the **command line**! So, don't click on it but rather run it via the Windows command line (remember DIS08).

Good luck.

Philipp Schaer

**Reply by ls0218** [reply]
Sorry, that makes a lot of sense. Thank you.

**Comment by breuert** [comment]
Dear Ms. Schulz,

Thank you for the feedback! We will have a look at how to run `trec_eval` with Windows on Friday! Of course, we can also recap the entire workflow. I will try to have a running VM with Win10, but maybe you could also prepare your environment for screen sharing.

**Reply by ls0218** [reply]
Dear Mr. Breuer,

thank you very much. That would be really helpful. I will prepare my environment for screen sharing.
